packages/block-editor/README.md

```diff
@@ -658,7 +658,10 @@ _Returns_
 
 ### useBlockEditContext
 
-Undocumented declaration.
+The `useBlockEditContext` hook provides information about the block this hook is being used in. 
```

Comment:
This part of the README file is autogenerated from the source code. You need to put the same description near the definition which I belive would be in:

https://github.com/WordPress/gutenberg/blob/5fc2e0f64d2b663deea8a7e1f9ecd82190d4ba10/packages/block-editor/src/components/block-edit/index.js#L12

Reply:
Done  👍 the text in the readme is now getting auto generated by the text above the export statement